docker-compose.yaml

```diff
@@ -0,0 +1,53 @@
+version: '3.1'
+
+services:
+
+  db:
+    image: mysql:5.7
```

Comment:
Instead of mysql we could also use mariadb which is a dropin for mysql which most hosting companies use.

Reply:
Sounds good to me. I updated the db image to use mariadb.